.ci/config/matrices.yaml

```
@@ -0,0 +1,76 @@
+# Data structure:
+#
+# $matrixId:
+#   stages: [LIST; REQUIRED; Doc: defines the stage types to be used in the matrix generation ]
+#   databases: [LIST; OPTIONAL; Doc: defines the databases to be used in the stage generation ]
+main-unit:
+  databases:
+  - 'postgresql_96'
+  - 'postgresql_94'
+  - 'postgresql_107'
+  - 'postgresql_112'
+  - 'postgresql_122'
+  - 'aws_aurora_postgresql_96'
+  - 'aws_aurora_postgresql_107'
+  - 'mariadb_100'
+  - 'mariadb_102'
+  - 'mariadb_103'
+  - 'mariadb_galera'
+  - 'mysql_57'
+  - 'oracle_11'
+  - 'oracle_12'
+  - 'oracle_18'
+  - 'oracle_19'
+  - 'db2_105'
+  - 'db2_111'
+  - 'sqlserver_2012'
+  - 'sqlserver_2016'
+  - 'sqlserver_2017'
+  - 'sqlserver_2019'
+  stages: 
+  - 'engine-unit'
+  - 'engine-unit-authorizations'
+  - 'webapp-unit'
+  - 'webapp-unit-authorizations'
+main-rest:
+  stages:
+  - 'engine-rest-unit-jersey-2'
+  - 'engine-rest-unit-resteasy'
+  - 'engine-rest-unit-resteasy3'
+  - 'engine-rest-unit-cxf'
+  - 'engine-rest-unit-wink'
+  - 'engine-rest-unit-jaxrs2-jersey-2'
+  - 'engine-rest-unit-jaxrs2-resteasy3'
+  - 'engine-rest-unit-compatibility-wildfly'
+  - 'engine-rest-IT-embedded-wildfly'
+daily-db:
```

Not sure about this one:
```suggestion
daily-stages:
```